### ISSUE TITLE
[Bug]: AoT About and Support page buttons don't show enough of a focus indicator

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typehero/typehero#1682]
### What happened?

When an element has focus, such as from <kbd>Tab</kbd> focus, that should be visually apparent: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html. Otherwise folks who are using a keyboard or other non-mouse device can't easily tell what they've currently got focused.

That focus indication needs to have enough color contrast that low vision folks can see it: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

The buttons in About and Support override the default focus styles in a way that's super subtle, and for at least the donation button, definitely not AA compliant:

https://github.com/user-attachments/assets/b034373d-87f0-4d9b-a9ad-850e9ab73906

### What browsers are you seeing the problem on?

All

### Relevant log output

```zsh
Aside: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html is a _beautiful_ thing for showing focus indications only when the user probably wants them.
```
